strawberryfields/backends/fockbackend/ops.py

```diff
@@ -390,11 +379,11 @@ def fockState(n, trunc):
 
 
 @functools.lru_cache()
-def coherentState(alpha, trunc):
+def coherentState(r, phi, trunc):
```

Comment:
Might be nice to reflect the change in the formula contained in the docstring as well :)